openpype/hosts/flame/startup/openpype_in_flame.py

```diff
@@ -3,16 +3,16 @@
 from Qt import QtWidgets
 from pprint import pformat
 import atexit
-import openpype
+
 import avalon
 import openpype.hosts.flame.api as opfapi
+from openpype.pipeline import install_host
 
 
 def openpype_install():
     """Registering OpenPype in context
     """
-    openpype.install()
-    avalon.api.install(opfapi)
+    install_host(opfapi)
     print("Avalon registered hosts: {}".format(
         avalon.api.registered_host()))
```

Comment:
This file should imported `from openpype.pipeline import registered_host` and change this line:
```suggestion
    print("Registered hosts: {}".format(registered_host()))
```

It seems `avalon.api.registered_hosts()` is used in a lot more areas of the code and would still need fixing?